Add load formulation

josephmckinsey opened the pull request:
This completes #61

Commit:
Add load formulation

scdhulipala reviewed (changes requested):
LGTM. Can you change the tests to include load formulation?

Commit:
Add testing and export new formulations

- Weird notes:
    There was a strange part where I couldn't get_components all loads,
    because the RTS systems has some loads with the same names as buses
    I think.

    I also had a very strange error when checking the load values.
    The PRAS load was off by 1 (integer rounding happened). It turns
    out this was a floating point error, amplified by the
    floor function.

- The recently added formulations were not a part of the export list in
    `src/SiennaPRASInterface.jl`

josephmckinsey commented:
> LGTM. Can you change the tests to include load formulation?

Fixed. It ended up being a bit of work to get it really tested. I should have done it from the start.

jd-lara reviewed (approved):
I think this is good to merge
